packages/nextjs/src/app-beta/client/ClerkProvider.tsx

```diff
@@ -1,5 +1,6 @@
 'use client';
 // !!! Note the import from react
+import type { ClerkProviderProps } from '@clerk/clerk-react';
 import { ClerkProvider as ReactClerkProvider } from '@clerk/clerk-react';
```

Comment:
@desiprisg Any ideas why this still comes from clerk/react and not clerk/nextjs?

Reply:
We probably can reuse it. I'll try to clean it up.

Reply:
Actually, we can't right now since the `useRouter` hook needs to be imported from `next/navigation` and not `next/router`, like it's done here.https://github.com/clerkinc/javascript/blob/46fcea92c69ef052eeebd54c5adf9ef82272e3fd/packages/nextjs/src/client/index.tsx#L4